File: test-output/exports-smoke/MCP_Workflow_Smoke_20251103_105903.api.py
# API replay script (smoke)
import json
servers = []
steps = []
__MCP_export_results__ = []
with open('MCP_Workflow_Smoke_20251103_105903.exported.json', 'w', encoding='utf-8') as f:
    f.write(json.dumps({'exported_at': __import__('datetime').datetime.utcnow().isoformat() + 'Z', 'servers': servers, 'steps': __MCP_export_results__}, ensure_ascii=False, indent=2))
print('SMOKE: wrote MCP_Workflow_Smoke_20251103_105903.exported.json')

File: test-output/exports-smoke/MCP_Workflow_Smoke_20251103_105903.api.js
// API replay script (smoke)
const fs = require('fs');
(async function(){
  const base = process.argv[2] || 'http://localhost:3060';
  const servers = [];
  const steps = [];
  const __MCP_export_results__ = [];
  // No sessions to open (servers = [])
  // No calls to perform (steps = [])
  try {
    const out = { exported_at: new Date().toISOString(), servers, steps: __MCP_export_results__ };
    fs.writeFileSync('MCP_Workflow_Smoke_20251103_105903.exported.json', JSON.stringify(out, null, 2));
    if (__MCP_export_results__.length) console.log(JSON.stringify(__MCP_export_results__[__MCP_export_results__.length-1].result, null, 2));
    console.log('SMOKE: wrote MCP_Workflow_Smoke_20251103_105903.exported.json');
  } catch (e) { console.error('SMOKE: failed to write export JSON', e); process.exit(3); }
})();
